src/Model/Segno.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace MvLabs\Chocosite\Model;
+//la classe è costruita per determinare se il magazzino deve essere caricato o scaricato
+//inoltre per evitare che il magazzino venga movimentato in modo improprio (vedi setGiacenza in db,php)
+class Segno {
```

Comment:
L'idea di questa classe è buona, anche se forse è un po' eccessiva. In caso la vedrei più utile se al posto di `+` e `-` utilizzassi delle costanti di classe (che però forse non abbiamo mai visto durante il corso) e ti assicurassi che l'input del costruttore fosse effettivamente uno dei valori attesi, tirando un'eccezione in caso contrario